src/Api/Controllers/AppsController.php

```diff
@@ -43,6 +56,27 @@ public function onConstruct()
         ];
     }
 
+    /**
+     * Process the create request and records the object.
+     *
+     * @return ModelInterface
+     *
+     * @throws Exception
+     */
+    protected function processCreate(RequestInterface $request) : ModelInterface
+    {
+        //process the input
+        $request = $this->processInput($request->getPostData());
+
+        if (array_key_exists('settings', $request)) {
+            $this->model->setSettings(json_decode($request['settings'], true));
```

Comment:
@rwhite27 check if this is a json using isJson and that it sorry I missed this before